sdk/core/azure-core-test/src/test_proxy_policy.cpp

```diff
@@ -42,14 +42,21 @@ std::unique_ptr<RawResponse> TestProxyPolicy::Send(
   {
     // This is a download with keep connection open. Let's switch the request
     redirectRequest = Azure::Core::Http::Request(
-        request.GetMethod(), Azure::Core::Url(m_testProxy->GetTestProxy()), false);
+        request.GetMethod(),
+        Azure::Core::Url(m_testProxy->GetTestProxy()),
+        request.GetBodyStream(),
+        false);
   }
 
   redirectRequest.GetUrl().SetPath(request.GetUrl().GetPath());
 
   // Copy all headers
   for (auto& header : request.GetHeaders())
   {
+    if (header.first == "host")
```

Comment:
Minor style nitpicking: In general, I would prefer to see something like:

```cpp
if (header.first != "host")
{
  redirectRequest.SetHeader(header.first, header.second);
}
```
simply because (to me) it makes the intent of the change clearer.
```

Reply:
Ok